sdk/storage/Azure.Storage.DataMovement.Blobs/tests/ProgressHandlerTests.cs

```diff
@@ -296,7 +296,7 @@ await TransferAndAssertProgress(
                 10 /* fileCount */,
                 transferManagerOptions: transferManagerOptions,
                 transferOptions: transferOptions,
-                waitTime: 30);
+                waitTime: 90);
```

Comment:
This was really needed?

Reply:
Actually yeah, so I was running this in Live mode and it looks like there seems to be a slowdown in acquiring the authentication token many times for small chunks. We can take this offline.